gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsInstrumentation.java

```diff
@@ -127,6 +125,11 @@ public class GhfsInstrumentation
 
   private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();
 
+  /**
+   * Instance of StatusStatisticListener for registering and update of statuscode related metrics
+   */
+  private static StatusStatisticListener STATUS_STATISTIC_INSTANCE;
```

Comment:
Currently we are not collecting any statistic other than counters. So, consider making this a bit more generic. Maybe rename to GcsIOMetrics